sale_order_partner_invoice_period/views/sale_order_view.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_order_form_inherit" model="ir.ui.view">
```

Comment:
Don't change XML ID from original, better preserve this:
```suggestion
    <record id="view_order_form" model="ir.ui.view">
```
This view will be actually fully named by Odoo as `sale_order_partner_invoice_period.view_order_form`, so its ID is truly unique, and in this way is simpler to find.